Digipost/src/main/res/layout/activity_notification_settings.xml

```diff
@@ -92,6 +92,14 @@
                     android:orientation="horizontal"
                     android:layout_marginBottom="80dp">
 
+                    <TextView
+                      android:id="@+id/notification_settings_countrycode_prefix"
+                      android:layout_width="wrap_content"
+                      android:layout_height="wrap_content"
+                      android:layout_marginRight="-4sp"
```

Comment:
For bedre støtte med Right-to-left:
```suggestion
                      android:layout_marginEnd="-4sp"
```

Reply:
Dette bør da også gjøres på textViewet på linje 82.